tests/acceptance/features/apiGraph/getGroup.feature

```diff
@@ -132,3 +132,29 @@ Feature: get groups and their members
       }
     }
     """
+
+
+  Scenario: Get details of group with UTF-8 characters name
+    Given group "$x<=>[y*z^2+1]!" has been created
```

Comment:
IMO, here UTF-8 really means the Unicode characters
```suggestion
    Given group "<some-unicode-chars>" has been created
```
CC @individual-it @ScharfViktor